### PR TITLE
Update- security

### PR DESCRIPTION
**Pull request by WT-Liu**
### What problem does this PR solve?

This is related to https://github.com/tikv/tikv/pull/7792

### What is changed and how it works?

update security and TiKV logos

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 
- No release note

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+For known public security vulnerabilities, we will disclose the disclosure as soon as possible after receiving the report. Vulnerabilities discovered for the first time will be disclosed in accordance with the following process:
+1. The received security vulnerability report shall be handed over to the security team for follow-up coordination and repair work.
+2. After the vulnerability is confirmed, we will create a draft Security Advisory on Github that lists the details of the vulnerability.
+3. Invite related personnel to discuss about the fix
+4. Fork the temporary private repository on Github, and collaborate to fix the vulnerability
+5. After the fix code is merged into all supported versions, the vulnerability will be publicly posted in the GitHub Advisory Database
+	
+## PGP Public Key
+
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Comment (dcalvin) [inline SECURITY.md:43]:
```suggestion
···
-----BEGIN PGP PUBLIC KEY BLOCK-----
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Cvsf/45LtqEYY/NOHCZVqmZmF8RqTdwrcH5f1EP31J+F9IoVgonw+4VAUCqBiQQk
+2MGR4XBASTYbJ8v8b8gzf2R0HZgvrDXxz1nwEaikmU7W+0r2RDGQy+etfJsarwkS
+qwmc1AiznSTiuu1x9iV44tnCcmdFo3YFcd8kRHr1aIy1kdaXCL+Lns5HLmKKfOjQ
+j9bAH01FuSSlPLoaTWoqf+Sm3Fu/KeW0mQHulDXujV1lYX0jUxy7lmj2P/yR8ZdD
+12o9srqrh4Rja72g5utmA7ZTuSq4YwsXvQ13c3agZKO+WuiQkRfsgkh/NIWTyCXn
+IvCICV7zG1cyuM/Z2Y7/TJ+upvahP46nM3s3G15b8FYuTSmRN1Kp9+mBt2BHqOy1
+5MKKdj+J5co/zuRpgsuIIgTDrkS0UjN57BsHbIXhTQmQeablbIaMUtIxCH8t0V3K
+ulx+VF4Lf9n3ydf593Nha9bMJ/rnSp01
+=XbYK
+-----END PGP PUBLIC KEY BLOCK-----
```
Comment (dcalvin) [inline SECURITY.md:94]:
```suggestion
-----END PGP PUBLIC KEY BLOCK-----
···
```

**Comment by BusyJay** [comment]
So what's the difference between it and #7792?

**Comment by WT-Liu** [comment]
> So what's the difference between it and #7792?

I addressed the comments you made in https://github.com/tikv/tikv/pull/7792 and updated the TiKV logo in readme. Sorry that I think I made a mistake and couldn't make a commit to https://github.com/tikv/tikv/pull/7792, so I opened a new one.

**Pull request merged**